appstudio-controller/controllers/appstudio.redhat.com/devsandbox_deployment_controller.go

```diff
@@ -34,6 +34,11 @@ import (
 	"sigs.k8s.io/controller-runtime/pkg/predicate"
 )
 
+const (
+	// Indicates the name of the provisioner that provisioned the DT.
+	annDynamicallyProvisioned = "provisioner.appstudio.redhat.com/provisioned-by"
```

Comment:
Any objections to moving this field into `application-api` repo, so it can be consumed by other teams?